persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalFileConstants.java

```diff
@@ -232,6 +232,11 @@ private InheritedTypeReferenceConstants() {
             " <%sRelations[]>self.include);";
     public static final String PLACEHOLDER_PATTERN = "\"\\$\\{(.*?)}\"";
     public static final String MYSQL_DRIVER_IMPORT = "import ballerinax/mysql.driver as _;";
+
+    public static final String AUTOGENERATED_FILE_COMMENT = "// AUTO-GENERATED FILE. DO NOT MODIFY.\n" +
```

Comment:
From the code looks like you take the system line separator into account. Would that be not applicable to multi line comments embedded here? Need to check this in all other places you have used it.

Reply:
updated in [4daf0c6](https://github.com/ballerina-platform/persist-tools/pull/79/commits/4daf0c6807a98caf258c2ef37d22f7361f78dad1)